src/workloads/execution/ColumnstoreIndexes.yml

```diff
@@ -0,0 +1,495 @@
+      {"$project": {"value": "$__alias_0", "_id": 0}},
+      {"$limit": 5000}
+    ]
+  allowDiskUse: true
+  cursor: {batchSize: 2000}
+
+ActiveCloudUsersLastMonthQuery: &ActiveCloudUsersLastMonthQuery
+  aggregate: *Collection
+  pipeline:
+    [
+      {"$addFields": {"target": 3000}},
+      {
+        "$match": {
+          "metadata.target": {"$in": ["cloud"]},
+          "created_at": {"$gte": ISODate("2020-03-09T01:10:04Z")},
+          "resource": {"$in": ["Dashboard"]}
+        }
+      },
+      {"$group": {"_id": {}, "__alias_0": {"$addToSet": "$user_id"}, "__alias_1": {"$min": "$target"}}},
+      {"$project": {"_id": 0, "__alias_0": {"$size": "$__alias_0"}, "__alias_1": 1}},
+      {"$project": {"value": "$__alias_0", "target": "$__alias_1", "_id": 0}},
+      {"$limit": 5000}
+    ]
+  allowDiskUse: true
+  cursor: {batchSize: 2000}
+
+VersionsHistogramQuery: &VersionsHistogramQuery
+  aggregate: *Collection
+  pipeline:
+    [
+      {"$match": {"metadata.target": {"$exists": true}}},
+      {
+        "$match": {
+          "created_at": {"$gte": ISODate("2020-04-02T01:10:52Z")},
+          "metadata.version": {"$nin": [null]},
+          "resource": {"$nin": ["Error"]}
+        }
+      },
+      {
+        "$group": {
+          "_id": {"__alias_0": "$metadata.version", "__alias_1": "$metadata.target"},
+          "__alias_2": {"$sum": 1}
+        }
+      },
+      {"$project": {"_id": 0, "__alias_0": "$_id.__alias_0", "__alias_1": "$_id.__alias_1", "__alias_2": 1}},
+      {"$project": {"x": "$__alias_0", "y": "$__alias_2", "color": "$__alias_1", "_id": 0}},
+      {"$limit": 50000}
+    ]
+  allowDiskUse: true
+  cursor: {batchSize: 2000}
+
+UniqueUsersPerMonthQuery: &UniqueUsersPerMonthQuery
+  aggregate: *Collection
+  pipeline:
+    [
+      {
+        "$match": {
+          "resource": {"$in": ["Dashboard"]},
+          "created_at": {"$gte": ISODate("2019-07-09T01:12:32Z")},
+          "metadata.target": {"$nin": [null, ""]}
+        }
+      },
+      {
+        "$addFields": {
+          "created_at": {
+            "$cond": {
+              "if": {"$eq": [{"$type": "$created_at"}, "date"]},
+              "then": "$created_at",
+              "else": null
+            }
+          }
+        }
+      },
+      {
+        "$addFields": {
+          "__alias_0": {
+            "year": {"$year": "$created_at"},
+            "month": {"$subtract": [{"$month": "$created_at"}, 1]}
+          }
+        }
+      },
+      {
+        "$group": {
+          "_id": {"__alias_0": "$__alias_0", "__alias_1": "$metadata.target"},
+          "__alias_2": {"$addToSet": "$user_id"}
+        }
+      },
+      {
+        "$project": {
+          "_id": 0,
+          "__alias_0": "$_id.__alias_0",
+          "__alias_1": "$_id.__alias_1",
+          "__alias_2": {
+            "$size": "$__alias_2"
+          }
+        }
+      },
+      {"$project": {"x": "$__alias_0", "y": "$__alias_2", "color": "$__alias_1", "_id": 0}},
```

Comment:
[nit] any reason for the odd formatting here?

Reply:
I'm not sure what you mean by "odd formatting". If you meant why a separate line for $project, then my answer would be "to reduce number of lines" if a stage can be fit into a line.